src/main/menus/marktext.js

```diff
@@ -1,47 +1,46 @@
+import { app } from 'electron'
 import * as actions from '../actions/marktext'
 
-export default function markTextMenu ({ app }) {
-  return {
-    label: 'Mark Text',
-    submenu: [{
-      label: 'About Mark Text',
-      role: 'about'
-    }, {
-      label: 'Check for updates...',
-      click (menuItem, browserWindow) {
-        actions.checkUpdates(menuItem, browserWindow)
-      }
-    }, {
-      label: 'Preferences',
-      accelerator: 'Cmd+,',
-      click (menuItem, browserWindow) {
-        actions.userSetting(menuItem, browserWindow)
-      }
-    }, {
-      type: 'separator'
-    }, {
-      label: 'Services',
-      role: 'services',
-      submenu: []
-    }, {
-      type: 'separator'
-    }, {
-      label: 'Hide Mark Text',
-      accelerator: 'Command+H',
-      role: 'hide'
-    }, {
-      label: 'Hide Others',
-      accelerator: 'Command+Alt+H',
-      role: 'hideothers'
-    }, {
-      label: 'Show All',
-      role: 'unhide'
-    }, {
-      type: 'separator'
-    }, {
-      label: 'Quit',
-      accelerator: 'Command+Q',
-      click: app.quit
-    }]
-  }
+export default {
+  label: 'Mark Text',
+  submenu: [{
+    label: 'About Mark Text',
+    role: 'about'
```

Comment:
Use `showAboutDialog` ?

Reply:
This menu is only for OSX, so should we use the linux/windows about dialog?

Reply:
So I suggest use linux/windows about dialog,

Reply:
Currently the "native" dialog is shown on OSX and the "custom" about dialog on linux/windows

Reply:
I suggest all system use the custom about dialog